test/test-function.js

```diff
@@ -0,0 +1,12 @@
+
+function testFunction(test, actualNumber) {
```

Comment:
you wrote this function and never used it! Using this function is part of the assignment instructions, so I recommend you refactor to use this function. In the future, if you have unused code like this, make sure it's deleted before you submit so that your code is clean.